pennylane/plugins/default_gaussian.py

```diff
@@ -651,14 +651,14 @@ def poly_quad_expectations(mu, cov, wires, params, hbar=2.):
     ex = np.trace(A @ cov) + k2
     var = 2*np.trace(A @ cov @ A @ cov) + d2.T @ cov @ d2
 
-    modes = np.arange(2*N).reshape(2, -1).T
+    modes = np.arange(2*total_wires).reshape(2, -1).T
```

Comment:
What happens here if `total_wires=None` which is the default argument?

Reply:
This will never be the case, since the plugin always calls these methods with `total_wires=self.num_wires`. I could make it a positional argument?

Reply:
The reason I didn't was to avoid breaking a majority of the plugin tests, etc.

Reply:
I would prefer having it as a positional argument because when I read the code right now it suggests that `None` is an acceptable parameter. Fixing the tests should not be that big of an issue, no? Just inserting 1s and 2s where it fails.

Reply:
Done, made it a positional argument :+1: